pkg/domain/infra/tunnel/images.go

```diff
@@ -270,7 +270,7 @@ func (ir *ImageEngine) Save(ctx context.Context, nameOrID string, tags []string,
 			defer func() { _ = os.Remove(f.Name()) }()
 		}
 	default:
-		f, err = os.Create(opts.Output)
+		f, err = os.OpenFile(opts.Output, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
```

Comment:
Can we add a comment here that this mode is important.

Reply:
Sure, could you explain to me why? Or what should be in the comment?


       default:
		// This code was added to allow for opening stdout use on MAC. The mode 0644 is important.
		f, err = os.OpenFile(opts.Output, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
	}

Reply:
I mean the flags os.O_WRONLY. If someone has to rework this they might not think about it and change it back to os.Create() for example.
Maybe adding a link to the issue would help

Reply:
Ok, I will write that.